apps/storybook-react/.storybook/main.ts

```diff
@@ -31,6 +31,10 @@ const config: StorybookConfig = {
     name: getAbsolutePath('@storybook/react-vite'),
     options: {},
   },
+  features: {
+    // Enable development mode for better async component support
+    developmentModeForBuild: true,
+  },
```

Comment:
After further reading this feature is only useful for async components so it can be removed for now https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#introducing-featuresdevelopmentmodeforbuild

```suggestion
```

Reply:
Removed